frontend/src/components/general/SocialMediaButton.js

```diff
@@ -16,8 +17,17 @@ const useStyles = makeStyles((theme) => ({
   }),
 }));
 
-export default function SocialMediaButton({ href, socialMediaIcon, altText, isFooterIcon }) {
-  const classes = useStyles({ isFooterIcon: isFooterIcon });
+export default function SocialMediaButton({
```

Comment:
The naming "Button" is a bit counter-intuitive here since it seems to just be a link

Reply:
It's really more of an iconbutton so perhaps we could use that for the name? i.e. SocialMediaIconButton